openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -23,6 +23,10 @@
     EMAIL_MIN_LENGTH, EMAIL_MAX_LENGTH, PASSWORD_MIN_LENGTH, PASSWORD_MAX_LENGTH,
     USERNAME_MIN_LENGTH, USERNAME_MAX_LENGTH
 )
+from .helpers import (
+    get_profile_image_names as _get_profile_image_names,
```

Comment:
Perhaps move these methods up to the helpers.py that is a peer of the accounts directory? And then have profile_images be a peer directory to accounts in user_api?